### ISSUE TITLE
Bug in token refresh

### ISSUE THREAD

**Issue by kamicut** [developmentseed/scoreboard#510]
There seems to be a bug in the scoreboard token refresh code when asking for a new teams token. We get this from hydra logs:

```sh
hydra_1  | time="2020-02-20T22:10:48Z" level=error msg="An error occurred" debug=": not_found" description="The request is missing a required parameter, includes an invalid parameter value, includes a parameter more than once, or is otherwise malformed" error=invalid_request hint="Make sure that the various parameters are correct, be aware of case sensitivity and trim your parameters. Make sure that the client you are using has exactly whitelisted the redirect_uri you specified."
```

**Comment by guidorice** [comment]
@kamicut @LanesGood I am also seeing  inconsistent behavior after leaving Scoreboard running for anywhere between an hour to overnight. I suspect it's caused by this bug.  (data, like # of teams, just goes to zero).

But also UI does not indicate the next steps user needs to take, like sign in again.  Only after going back to osm-teams, reloading that  app, and then reloading Scoreboard app, do I see my teams listing again.

I think is a high priority issue- much higher than adding any new features to Scoreboard.

**Comment by guidorice** [comment]
I noticed both scoreboard and osm-teams are behind 1 major version in the `simple-oauth2` package. I will fix that in in #515 .
